src/components/ChoosePin.js

```diff
@@ -36,13 +37,23 @@ class ChoosePin extends React.Component {
     this.props.updatePin(value);
   }
 
+  /**
+   * Check API version and update server network before loading data
+   */
+  handleSuccess = () => {
```

Comment:
Isn't it better to add it to the reconnect event? I feel it is more related to each action instead of the act of setting a PIN.

Reply:
Yeah, it's better. Changed it, thanks